module/mempool/stdmap/backDataHeapBenchmark_test.go

```diff
@@ -193,14 +193,47 @@ func (b baselineLRU) All() map[flow.Identifier]flow.Entity {
 
 		entity, ok := b.ByID(id)
 		if !ok {
-			panic("could not retrive entity from mempool")
+			panic("could not retrieve entity from mempool")
 		}
 		all[id] = entity
 	}
 
 	return all
 }
 
+func (b baselineLRU) Identifiers() flow.IdentifierList {
+	ids := make(flow.IdentifierList, b.c.Len())
+	i := 0
+	for _, entityID := range b.c.Keys() {
+		id, ok := entityID.(flow.Identifier)
+		if !ok {
+			panic("could not assert to entity id")
+		}
+		ids[i] = id
+		i++
+	}
+	return ids
+}
+
+func (b baselineLRU) Entities() []flow.Entity {
+	entities := make([]flow.Entity, b.c.Len())
+	i := 0
+	for _, entityID := range b.c.Keys() {
```

Comment:
See above comment.  Let's replace 'range' with a 'for' loop.

Reply:
https://github.com/onflow/flow-go/pull/1852/commits/f2768318602b479b6a3ad0f9c96db6370433310b